### PR TITLE
Add option to disable banner in console

### PR DESCRIPTION
**Pull request by hfjallemark**

### CODE REVIEW

File: lib/assets/javascripts/unpoly/log.coffee [hunk 1]
```diff
@@ -37,12 +37,15 @@ up.log = do ->
     prints to the developer console.
   @param {string} [options.prefix='[UP] ']
     A string to prepend to Unpoly's logging messages so you can distinguish it from your own messages.
+  @param {boolean} [options.banner=true]
+    Whether to show the unpoly banner in the console.
```
Comment (adam12) [inline lib/assets/javascripts/unpoly/log.coffee:41]:
```suggestion
    Print the Unpoly banner to the developer console.
```

**Review by adam12 — COMMENTED** [review]
Looks good but I'd suggest changing the messaging slightly to better align with the rest of the option descriptions.

**Comment by hfjallemark** [comment]
> Looks good but I'd suggest changing the messaging slightly to better align with the rest of the option descriptions.

Done @adam12.

**Review by adam12 — APPROVED** [review]
:+1: LGTM but I can't build locally to test (cc @triskweline)

**Review by triskweline — APPROVED** [review]

**Pull request merged**